create.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <title>Create!</title>
+  <link rel="stylesheet" type="text/css" href="temp.css" >
```

Comment:
temp is not a good name for any file that makes it into the master branch
make sure you name your files after what they actually contain :)